lib/services/groups/groupRegistryMemory.js

```diff
@@ -87,6 +87,40 @@ function getConfigurationsByService(service) {
     });
 }
 
+/**
+ * List all the configGroups created in the IoT Agent. The result, passed as a parameter in the callback
+ * will be an object with two attributes: the services array with the list of groups; and a count
+ * attribute with the total number of groups in the collection.
+ *
+ * @param {Number} service      Service for wich all the configurations want to be retrieved.
+ * @param {Number} limit        Maximum number of entries to return.
+ * @param {Number} offset       Number of entries to skip for pagination.
+ */
+function listConfigGroups(service, limit, offset, callback) {
+    const result = [];
+    let skipped = 0;
+    const filteredGroups = getConfigurationsByService(service);
+
+    for (const i in filteredGroups) {
+        if (registeredGroups.hasOwnProperty(filteredGroups[i])) {
+            if (offset && skipped < parseInt(offset, 10)) {
+                skipped++;
+            } else {
+                result.push(registeredGroups[filteredGroups[i]]);
+            }
+
+            if (limit && result.length === parseInt(limit, 10)) {
+                break;
+            }
+        }
+    }
+
+    callback(null, {
+        count: filteredGroups.length,
+        configGroups: result
+    });
+}
+
```

Comment:
This is duplicated, same as `function listGroups(service, limit, offset, callback) `. No needed to duplicate code

Reply:
Fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/1375/commits/11787050de050581b6b76221f20d9c2c671c3b11